_pages/en/strategy-build-vs-buy.md

```diff
@@ -203,18 +293,18 @@ The requirement is that the system level architectural decisions and expertise r
 In contrast to traditional build decisions[^2], when the Government of Canada purchases resources from industry the Government of Canada will be the owner of the product itself -- responsible for the projects implementation and operations; overseeing teams supplied by vendors who provide the technical expertise[^3].This ensures that knowledge about the details and functioning of the system exist within the department[^4].
 Expertise and knowledge regarding implementation details of systems Canadians rely on must exist within the Government of Canada[^5].
 
-#### Alignment Actions
+#### Alignment Actions<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).